src/main/java/frc/utils/auto/PathPlannerUtils.java

```diff
@@ -47,49 +71,46 @@ public static void scheduleWarmup() {
 		PathfindingCommand.warmupCommand().schedule();
 	}
 
-
-	private static void reportAlert(Alert.AlertType alertType, String message) {
-		new Alert(alertType, AutonomousConstants.LOG_PATH_PREFIX + "/" + message).report();
-	}
-
 	public static Optional<RobotConfig> getGuiRobotConfig() {
 		try {
 			RobotConfig robotConfig = RobotConfig.fromGUISettings();
 			return Optional.of(robotConfig);
 		} catch (IOException ioException) {
-			reportAlert(Alert.AlertType.ERROR, "GetGuiSettingsFailNotFoundAt");
+			reportAlert(Alert.AlertType.ERROR, "GuiSettingsFileNotFoundAt");
 		} catch (ParseException parseException) {
 			reportAlert(Alert.AlertType.ERROR, "GuiSettingsParseFailedAt");
 		}
 		return Optional.empty();
 	}
 
-	public static void configPathPlanner(
-		Supplier<Pose2d> poseSupplier,
-		Consumer<Pose2d> resetPose,
-		Supplier<ChassisSpeeds> robotRelativeSpeedsSupplier,
-		Consumer<ChassisSpeeds> robotRelativeSpeedsSetter,
-		PPHolonomicDriveController holonomicDriveController,
-		RobotConfig robotConfig,
-		BooleanSupplier shouldFlipPath,
-		GBSubsystem... driveRequirements
-	) {
-		AutoBuilder.configure(
-			poseSupplier,
-			resetPose,
-			robotRelativeSpeedsSupplier,
-			robotRelativeSpeedsSetter,
-			holonomicDriveController,
-			robotConfig,
-			shouldFlipPath,
-			driveRequirements
-		);
-	}
-
 	public static void registerCommand(String commandName, Command command) {
 		NamedCommands.registerCommand(commandName, command);
 	}
 
+	static Optional<PathPlannerPath> getPathFromFile(String pathName) {
+		try {
+			return Optional.of(PathPlannerPath.fromPathFile(pathName));
+		} catch (Exception exception) {
+			reportAlert(Alert.AlertType.ERROR, exception.getMessage());
+		}
+		return Optional.empty();
+	}
+
+	public static Pose2d getPathStartingPose(PathPlannerPath path) {
+		return new Pose2d(path.getPathPoses().get(0).getTranslation(), path.getIdealStartingState().rotation());
```

Comment:
```suggestion
		return new Pose2d(path.getPathPoses().get(0).getTranslation(), path.getIdealStartingState().rotation());
```

is getPathPoses().get(0) safe in all cases (e.g. path for a point to the same point)?

Reply:
it is impossible to create a path without a starting point